api/src/main/java/org/openmrs/api/AdministrationService.java

```diff
@@ -90,8 +91,11 @@ public interface AdministrationService extends OpenmrsService {
 	 * <strong>Should</strong> get property value given valid property name
 	 * <strong>Should</strong> get property in case insensitive way
 	 */
+	@Authorized(PrivilegeConstants.GET_GLOBAL_PROPERTIES)
 	public String getGlobalProperty(String propertyName);
 	
+	public String getPublicGlobalProperty(String propertyName);
```

Comment:
What does the above method do?

Reply:
We could use this method to grant access to essential global properties during startup.